projects/cashmere/src/lib/app-switcher/app-switcher.component.html

```diff
@@ -10,7 +10,7 @@ <h4>Applications</h4>
         <div class="apps">
             <div class="app" *ngFor="let app of applications">
                 <a class="thumbnail" title="{{app.Description}}" href="{{app.ServiceUrl}}" target="_blank">
-                    <div class="thumbnail-img" [style.background-image]="'url(' + app.icon + ')'"></div>
```

Comment:
I seem to remember the `style.background-image` being made that way as a fix for #503 .  Can you verify that this still works with the change?

Reply:
The discovery service returns a base64 image. I found this most recent change broke the discovery service integration. I have tested my fix and it seems to work

Reply:
hey @benjanderson - the main reason for using `background-img` on a div instead of an img was that it allows us to scale any image proportionally that's added to the app switcher.  With the switch as you've changed it, an image that isn't 1:1 will be distorted.  Do base64 images not work as a background-image?